docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -1210,6 +1213,65 @@ For more information on the relationship between the elements, see the [element
 <feature name="http://tizen.org/feature/fmradio">true</feature>
 ```
 
+### \<dependencies\> Element
+
+This element contains a set of packages which are have dependent to this Tizen package. The package can be installed when all dependencies are satisfied.
+
+For more information on the relationship between the elements, see the [element hierarchy](#hierarchy).
+
+**Occurrences:**
+
+- 1 (optional)
+
+**Expected children:**
+
+| Child element             | Occurrences          |
+|---------------------------|----------------------|
+| `<dependency>`            | 1 or more (optional) |
+
+**For example:**
+
+```xml
+<dependencies>
+   <dependency>.....</dependency>
+</dependencies>
+```
+
+#### \<dependency\> Element
+
+This element contains a required package for a Tizen package.
+
+**Occurrences:**
+
+- 1 or more (optional)
+
+**Attributes:**
+
+- `type`
+
+  The type of dependency (available values: "requires", "wants")
+
+  If the type is "requires", a Tizen package can be installed when the dependent package is installed
+
+  If the type is "wants", a Tizen package can be installed even if the dependent package is not installed
```

Comment:
These two sentences are complete sentences, hence, they require a period in the end. However, throughout the page, many places period is not used.

@mijin85cho Please give your valuable suggestion on this.

@jeremy-jang Please wait for @mijin85cho before updating.

Reply:
@rishu-samsung I checked the sentences in this page. As these sentences are in complete sentence form not like some other sentences, I think we can use periods at the end of these sentences.

Would you please add a period at the end of line 1254 and 1256?

Reply:
I added periods.